package.json

```diff
@@ -26,30 +26,31 @@
     "express": "^4.10.7",
     "glob": "^4.3.2",
     "http-proxy-middleware": "~0.11.0",
+    "ip": "~1.1.2",
```

Comment:
@geoff-reid It looks like there are some changes in here that aren't part of your patch.